build/CoreGraphics/lib/CoreGraphicsLib.vcxproj

```diff
@@ -22,7 +22,6 @@
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\CoreGraphics\CGBitmapImage.mm" />
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\CoreGraphics\CGColorSpace.mm" />
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\CoreGraphics\CGContext.mm" />
-    <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\CoreGraphics\CGContextCairo.mm" />
```

Comment:
gotcha

---
In reply to: [83311559](https://github.com/Microsoft/WinObjC/pull/1126#discussion_r83311559) [](ancestors = 83311559,83311217)